### PR TITLE
VB-4711 Re-implement DPS components using NPM module

### PR DESCRIPTION
**Pull request by tpmcgowan**
The DPS front-end components can now be incorporated using an NPM package (https://www.npmjs.com/package/@ministryofjustice/hmpps-connect-dps-components).

Replace existing implementation (which used the footer component only) with the module implementation.

This will simplify subsequently using the DPS header component.

### CODE REVIEW

File: assets/sass/application.scss [hunk 1]
```diff
@@ -7,11 +7,13 @@ $govuk-page-width: $moj-page-width;
 @import 'govuk/all';
 @import 'moj/all';
 
+@import 'node_modules/@ministryofjustice/hmpps-connect-dps-components/dist/assets/footer';
+// TODO enable: @import 'node_modules/@ministryofjustice/hmpps-connect-dps-components/dist/assets/header-bar';
```
Comment (tpmcgowan) [inline assets/sass/application.scss:11]: Have left the various header-related bits from following the set-up instructions temporarily commented out. Will be turning on in following PRs...

File: cypress.config.ts [hunk 2]
```diff
@@ -33,7 +32,6 @@ export default defineConfig({
         ...manageUsersApi,
         ...tokenVerification,
 
-        ...frontendComponents,
```
Comment (tpmcgowan) [inline cypress.config.ts:36]: Have removed wiremock stubs for front-end components. If defaults to using fallbacks when run in Cypress.

File: server/app.ts [hunk 3]
```diff
@@ -48,7 +50,14 @@ export default function createApp(services: Services): express.Application {
   app.use(setUpCurrentUser(services))
   app.use(appInsightsOperationId)
 
-  app.get('*', getFrontendComponents(services))
+  app.get(
+    '*',
+    dpsComponents.getPageComponents({
+      dpsUrl: config.dpsHome,
+      logger,
+      // TODO includeMeta: true,
```
Comment (tpmcgowan) [inline server/app.ts:58]: This will be turned on to get user caseload data, when header is pulled in.

**Review by hutcheonb-moj — APPROVED** [review]
LGTM

**Review by psoleckimoj — APPROVED** [review]
LGTM!

**Pull request merged**